pages/1.7/administration/release-notes/1-7/index.md

```diff
@@ -42,11 +42,9 @@ You can monitor the health of your cluster components from the DC/OS web interfa
 
 *   **Stateful applications using External Volumes (Experimental)** Consumption (attach & detach to containers) support in DC/OS for Amazon Web Services EBS, OpenStack Cinder, EMC Isilon, ScaleIO, VMAX, XtremeIO & Google Compute Engine). For more information, see the [documentation][5].
 
-# Fine-grained DC/OS access control
+# Fine-grained DC/OS access control  [enterprise type="inline" size="small"/]
```

Comment:
Should this be a different size tag? As an example in DC/OS 1.11 notes I found a heading tag that is formatted like this:
```
[enterprise]
### Security
[/enterprise]
```